src/lib/archwizard.module.ts

```diff
@@ -15,26 +15,9 @@ import {WizardCompletionStepDirective} from './directives/wizard-completion-step
 import {WizardStepSymbolDirective} from './directives/wizard-step-symbol.directive';
 import {WizardStepTitleDirective} from './directives/wizard-step-title.directive';
 import {WizardStepDirective} from './directives/wizard-step.directive';
-import {NAVIGATION_MODE_FACTORY, NavigationModeFactory} from './navigation/navigation-mode-factory.interface';
-import {BaseNavigationModeFactory} from './navigation/base-navigation-mode-factory.provider';
+import {NavigationModeDirective} from './directives/navigation-mode.directive';
 
 
-/**
- * Configuration object for the `angular-archwizard` module.
- *
- * Allows to customize global settings.
- */
-export interface ArchwizardModuleConfig {
-
-  /**
-   * Custom factory of [[NavigationMode]] instances.
-   *
-   * You may need a custom factory in order to support custom navigation modes.
-   * By default, [[BaseNavigationModeFactory]] is used.
-   */
-  navigationModeFactory?: NavigationModeFactory;
-}
-
```

Comment:
With this change, user loses ability to configure a default navigation mode globally (which was only present in the `develop` branch and never released). Is it needed?

Reply:
No I don't think it is needed. I think it is ok (even more common) to directly pass a navigation mode instance instead of passing a factory.